cfme/tests/control/test_actions.py

```diff
@@ -649,9 +647,6 @@ def create_one_snapshot(n):
             message="wait for snapshot %d to appear" % (n + 1),
             delay=5,
         )
-        current_snapshot = vm.current_snapshot_name
-        logger.debug("Current Snapshot Name: {}".format(current_snapshot))
```

Comment:
Q: any special reason to remove this logging and assertion?
Its checking `Active` state from `All Snapshot Tree`.

Other thought like not to assert thing in model page is better; but make sure we have test for this.

Reply:
This is due to one of the BZ's mentioned in the docblock https://bugzilla.redhat.com/show_bug.cgi?id=1745065

This BZ is not really relevant to this test case, but yeah I agree that we should have some automation to cover that BZ. I will work on a small test case to cover this.